pkg/agent/controller/networkpolicy/cache.go

```diff
@@ -689,6 +690,39 @@ func toRule(r *v1beta.NetworkPolicyRule, policy *v1beta.NetworkPolicy, maxPriori
 	return rule
 }
 
+// toStretchedNetworkPolicySecurityRule converts v1beta.NetworkPolicyRule to *rule
+// which is only used for security purpose of StretchedNetworkPolicy. The security
+// rule arbitrarily drops all traffic initiated from Pods with UnknownLabelIdentity.
+// Pods may have UnknownLabelIdentity when their Labels are completely new Labels
+// among the whole ClusterSet. Before the new allocated LabelIdentity is imported
+// in the Cluster, those Pods will have UnknownLabelIdentity.
+func toStretchedNetworkPolicySecurityRule(r *v1beta.NetworkPolicyRule, policy *v1beta.NetworkPolicy, maxPriority int32) *rule {
```

Comment:
This seems a little intrusive to the cache module which was just a pure data storage. Could we keep the stretched network policy logic in network_policy.go like we add the default drop for K8s Netpol?